test/fixtures/IL-2022.json

```diff
@@ -0,0 +1,290 @@
+[
+  {
+    "date": "2022-01-17 00:00:00 -0600",
+    "start": "2022-01-16T16:00:00.000Z",
+    "end": "2022-01-17T16:00:00.000Z",
+    "name": "Tu Bishvat (Arbor Day)",
+    "type": "observance",
+    "rule": "15 Shvat",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2022-02-14 00:00:00 -0600",
+    "start": "2022-02-13T16:00:00.000Z",
+    "end": "2022-02-14T16:00:00.000Z",
+    "name": "Fast of Esther",
+    "type": "school",
+    "rule": "13 Adar",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2022-02-15 00:00:00 -0600",
+    "start": "2022-02-14T16:00:00.000Z",
+    "end": "2022-02-16T16:00:00.000Z",
+    "name": "Purim",
+    "type": "school",
+    "note": "optional paid leave",
+    "rule": "14 Adar P2D",
+    "_weekday": "Tue"
+  },
+  {
+    "date": "2022-04-11 00:00:00 -0600",
+    "start": "2022-04-10T15:00:00.000Z",
+    "end": "2022-04-11T15:00:00.000Z",
+    "name": "Aliyah Day",
+    "type": "observance",
+    "note": "Official holiday, business as usual",
+    "rule": "10 Nisan",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2022-04-16 00:00:00 -0600",
+    "start": "2022-04-15T15:00:00.000Z",
+    "end": "2022-04-16T15:00:00.000Z",
+    "name": "Passover",
+    "type": "public",
+    "rule": "15 Nisan",
+    "_weekday": "Sat"
+  },
+  {
+    "date": "2022-04-17 00:00:00 -0600",
+    "start": "2022-04-16T15:00:00.000Z",
+    "end": "2022-04-21T15:00:00.000Z",
+    "name": "Passover (intermediate days)",
+    "type": "school",
+    "note": "School holiday, collective paid leave in many businesses and government offices",
+    "rule": "16 Nisan P5D",
+    "_weekday": "Sun"
+  },
+  {
+    "date": "2022-04-22 00:00:00 -0600",
+    "start": "2022-04-21T15:00:00.000Z",
+    "end": "2022-04-22T15:00:00.000Z",
+    "name": "Seventh day of Passover",
+    "type": "public",
+    "rule": "21 Nisan",
+    "_weekday": "Fri"
+  },
+  {
+    "date": "2022-04-23 00:00:00 -0600",
+    "start": "2022-04-22T15:00:00.000Z",
+    "end": "2022-04-23T15:00:00.000Z",
+    "name": "Mimouna",
+    "type": "observance",
+    "note": "Unpaid leave upon request",
+    "rule": "22 Nisan",
+    "_weekday": "Sat"
+  },
+  {
+    "date": "2022-04-28 00:00:00 -0600",
+    "start": "2022-04-27T15:00:00.000Z",
+    "end": "2022-04-28T15:00:00.000Z",
+    "name": "Holocaust Remembrance Day",
+    "type": "observance",
+    "note": "National remembrance day, business as usual except places of public entertainment",
+    "rule": "27 Nisan",
+    "_weekday": "Thu"
+  },
+  {
+    "date": "2022-05-05 00:00:00 -0600",
+    "start": "2022-05-04T15:00:00.000Z",
+    "end": "2022-05-05T15:00:00.000Z",
+    "name": "Fallen Soldiers and Victims of Terrorism Remembrance Day",
+    "type": "observance",
+    "note": "National remembrance day, business as usual except places of public entertainment",
+    "rule": "4 Iyyar",
+    "_weekday": "Thu"
+  },
+  {
+    "date": "2022-05-06 00:00:00 -0600",
```

Comment:
Independence Day (Yom HaAtzmaut) 2022 is on May 5

Reply:
Should be solved